lib/t/push_notifications/apns.ex

```diff
@@ -228,18 +228,10 @@ defmodule T.PushNotifications.APNS do
     base_alert_payload(type, alert, %{"user_id" => user_id})
   end
 
-  def build_alert_payload("live_mode_today" = type, _data) do
-    day_of_week = Date.utc_today() |> Date.day_of_week()
-
-    body =
-      case day_of_week do
-        6 -> dgettext("apns", "Come to the party at 20:00, it will be 🔥")
-        4 -> dgettext("apns", "Come to the party at 19:00, it will be 🔥")
-      end
-
+  def build_alert_payload("live_mode_today" = type, %{"time" => time}) do
     alert = %{
       "title" => dgettext("apns", "Since LIVE today 🥳"),
-      "body" => body
+      "body" => dgettext("apns", "Come to the party at %{time}, it will be 🔥", time: time)
```

Comment:
%{time} here is for the core TZ? so for user in the same region, but different TZ it will be misleading, right?

Reply:
moved to #461